src/librawspeed/metadata/CameraMetaData.h

```diff
@@ -23,18 +23,21 @@
 #include "common/Common.h" // for uint32
 #include <map>             // for map
 #include <string>          // for string
+#include <tuple>           // for tuple
 
 namespace RawSpeed {
 
 class Camera;
 
+using CameraId = std::tuple<std::string, std::string, std::string>;
+
```

Comment:
This obviously duplicates/colides with the work I did last night (see TiffID). I suggest to use a trivial struct like the one I introduced instead of a tuple because it is _way_ less likely to mix up the meaning of the individual members if they have a name (make/model/mode) instead of just an index in a tuple.

Maybe you could merge the two efforts if you moved/renamed my TiffID struct and added a mode string there. It would unify the whole 'id' related information?